trio/_core/_tests/test_asyncgen.py

```diff
@@ -242,13 +249,13 @@ async def step_outside_async_context(aiter):
     # NB: the strangeness with aiter being an attribute of abort_fn is
     # to make it as easy as possible to ensure we don't hang onto a
     # reference to aiter inside the guts of the run loop.
-    def abort_fn(_):
+    def abort_fn(_: object) -> _core.Abort:
         with pytest.raises(StopIteration, match="42"):
-            abort_fn.aiter.asend(None).send(None)
-        del abort_fn.aiter
+            abort_fn.aiter.asend(None).send(None)  # type: ignore[attr-defined]  # Callables don't have attribute "aiter"
+        del abort_fn.aiter  # type: ignore[attr-defined]
```

Comment:
why is the parameter `object` instead of `Callable[..., object]`?

Reply:
Because it's unused in the function body

Reply:
Changed in 9c09d80

Reply:
well, technically it is used - but I was mostly thinking about getting type errors on the callers side if they're passing something that isn't a `Callable`.